pkg/freeze/containerd/pause_test.go

```diff
@@ -120,14 +135,23 @@ func TestContainerResume(t *testing.T) {
 }
 
 func TestNoQueueProxyPause(t *testing.T) {
+	queueProxy := Container("queueproxy", "queue-proxy")
+
 	containers = []*cri.Container{&queueProxy}
 	results = []string{}
+	method = ""
 
-	f := FakeContainerdCri{}
-	cntr, _ := f.List(nil, nil, "")
-	ids, _ := lookupContainerIDs(cntr)
-	if !compareContainerLists(ids, results) {
-		t.Errorf("expected %s, got %s", ids, results)
+	f := FakeContainerdCri{
+		containers: containers,
+		results:    results,
+		method:     method,
+	}
+	_, err := f.List(nil, nil, "")
+	if err == nil {
+		t.Errorf("expecting error and got nil")
+	}
+	if err.Error() != "expected list of containers ids but got no non queue-proxy containers found in pod" {
+		t.Errorf("expecting \"no non queue-proxy\" error but got %v", err)
```

Comment:
can do this as:
```suggestion
		t.Errorf(`expecting "no non queue-proxy" error but got %v`, err)
```
or, probably better:
```suggestion
		t.Errorf(`expecting %q error but got %q`, "no non queue-proxy", err)
```